configs/default/GEPR-GEPRCF722_BT_HD.config

```diff
@@ -101,39 +101,49 @@ dma pin A01 0
 
 # feature
 feature -RX_PARALLEL_PWM
+feature -AIRMODE
 feature RX_SERIAL
 feature TELEMETRY
+feature LED_STRIP
 feature OSD
 feature ESC_SENSOR
 
 # serial
-serial 0 8192 115200 57600 0 115200
+serial 0 1 115200 57600 0 115200
 serial 1 64 115200 57600 0 115200
-serial 3 1 19200 57600 0 115200
-serial 4 1024 19200 57600 0 115200
+serial 3 1 115200 57600 0 115200
+serial 4 1024 115200 57600 0 115200
+
+# aux
+aux 0 0 0 1700 2100 0 0
+aux 1 1 1 1700 2100 0 0
+aux 2 28 1 900 1300 0 0
+aux 3 40 4 1700 2100 0 0
```

Comment:
What are these for?

Reply:
Hello @DusKing1 . These aux settings are mainly used to configure the use of Bluetooth and digital switches for user priority. So it should be kept here.